target/consensource-compose
# Reinit sds submodule

Signed-off-by: Patrick-Erichsen <Patrick.Erichsen@target.com>

## Proposed change/fix

Fixes a submodule issue with `sds` that was causing failures on init.

## Types of changes

What types of changes does this pull request introduce to ConsenSource? _Put an `x` in the boxes that apply_


## How to run/test

Run `git clone --recurse-submodules https://github.com/target/consensource-compose.git` and confirm that you don't receive any errors, and that the `sds` folder is populated.
